cherrypy/lib/static.py

```diff
@@ -28,6 +29,21 @@ def _setup_mimetypes():
 _setup_mimetypes()
 
 
+def _utf8_content_disposition(disposition, file_name):
+    """Create HTTP header for downloading a file with UTF-8 filename.
+
+    See this and related answers:
+    https://stackoverflow.com/a/8996249/2173868
+    """
+    ascii_name = unicodedata.normalize(
```

Comment:
Could you please change the style here to
```python
var_name = (
    multi().line().
    chain().of().calls()
)
```
?

Reply:
Yes, no problem.

Reply:
Yes, no problem.